_posts/2014-04-05-overview.md

```diff
@@ -5,30 +5,37 @@ permalink: overview
 github: "https://github.com/ember-cli/ember-cli.github.io/blob/master/_posts/2014-04-05-overview.md"
 ---
 
-Ember CLI is the [Ember.js](http://emberjs.com) command line utility that
-provides a fast [Broccoli](https://github.com/broccolijs/broccoli#broccoli)-powered
-asset pipeline, a strong conventional project structure, and a powerful [addon
-system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints)
-for extension. [Get started.](/user-guide/#getting-started)
+Ember CLI is the official command line utility used to create, build, serve, 
+and test [Ember.js](http://emberjs.com) apps and addons.
+
+It provides a consistent project structure, a powerful [addon
+system](http://www.ember-cli.com/extending/#developing-addons-and-blueprints), and
```

Comment:
Can you use https in the links?

Reply:
All links that can be https-ed are now changed to that. A few things have no https.